### DISCUSSION TITLE
Fail in Fedora 36

### DISCUSSION THREAD

**Discussion by Irbingou** [ohmyzsh/ohmyzsh#11051] [Help]
In fedora 36, it installs correctly but when restarting the terminal it shows as if it had not been installed :(

![Captura desde 2022-07-15 14-58-05](https://user-images.githubusercontent.com/51411412/179301871-c7dbef20-cb56-42c0-ab77-5c76a8324892.png)

**Comment by carlosala** [comment]
You probably need to change the default shell. Execute  `sudo lchsh $USER` and choose `zsh`. Then reboot and it should work!

**Reply by Irbingou** [reply]
Hi
Run command, reboot my machine and fedora boot from scratch, lost files, credentials :(

**Reply by carlosala** [reply]
I tried that in a VM and worked, really weird.

**Reply by Irbingou** [reply]
Nothing...
when executing sudo chsh $USER, I restarted my computer and these users or groups were created, how can I return to the user's
irving:x:1000:1000?

![imagen](https://user-images.githubusercontent.com/51411412/180069013-cc7a965f-618c-4eda-a206-0d89d20a0937.png)

**Reply by carlosala** [reply]
Did you executed `lchsh` or `chsh`?

**Reply by Irbingou** [reply]
chsh $USER
next I'm write zsh 🫢

**Reply by carlosala** [reply]
oh, the proper command is `lchsh`. Could you try in a Fedora's forum to see if someone knows how to come back to the previous state?
